psc-bundle/README.md

```diff
@@ -4,14 +4,15 @@ A dead code elimination tool for PureScript-style CommonJS modules. This can be
 
 ## Usage
 
-    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG]
-    
+    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG] [--optimize uncurry]
```

Comment:
This is confusing; the actual parser seems to only accept `--optimize`, rather than `--optimize uncurry` as stated here.

Should `optimize` actually take a list of options instead like this text suggests? Uncurrying, as far as I can tell, won't be the only optimisation.